pkg/processor/sourceprocessor/README.md

```diff
@@ -84,3 +84,35 @@ prefixed with the value defined in `keys.annotation_prefix` config option.
 This can be achieved with the [Kubernetes processor](../k8sprocessor).
 
 For example, if a resource has the `k8s.pod.annotation.sumologic.com/exclude` attribute, the resource will be dropped.
+
+### Container-level pod annotations
```

Comment:
Do we support source category prefix for container-level pod annotations?

Reply:
Later in the section:

```
No other transformations are applied to the source categories retrieved from container-level annotations, like adding source category prefix or replacing the dash.
```